Adyen/adyen-magento2
# Fixed getting cc type name

**Description**
I've found a bug during testing payments with credit cards. It keeps showing "Unknown" no matter what kind of test credit card was used.
![image](https://user-images.githubusercontent.com/36896606/176430616-74f9e5fa-f1b4-4e10-9a28-9e0e8309a923.png)

I did some debug and here is what I found. There is a method called `getAdyenCcTypes()` and it loads all cc types from adyen_pament.xml. `\Adyen\Payment\Block\Info\Cc::getCcTypeName` method uses this method and looks after valid ccType in    `$types` variable. This variable is an array with ccType keys, but in uppercase. So I think the right solution was to add change the line `\Adyen\Payment\Block\Info\Cc.php:30`

Here is the result.
![image](https://user-images.githubusercontent.com/36896606/176431628-e2c9fc14-0a5c-4636-8472-e8053daabe1e.png)

**Tested scenarios**
1. Create an order with credit card.
2. Check your order confirmation email or check your order on admin order page.
